src/chart.js

```diff
@@ -17,6 +17,7 @@ Chart.Element = require('./core/core.element');
 Chart.elements = require('./elements/index');
 Chart.Interaction = require('./core/core.interaction');
 Chart.layouts = require('./core/core.layouts');
+Chart.LinearScaleBase = require('./scales/scale.linearbase');
```

Comment:
Can we add a comment to remove this at v3 since it should be internal only?

Reply:
Done